src/ResourceManagement/Scheduler/Microsoft.Azure.Management.Scheduler/Generated/Models/BasicAuthentication.cs

```diff
@@ -33,7 +33,7 @@ public BasicAuthentication(HttpAuthenticationType? type = default(HttpAuthentica
         public string Username { get; set; }
```

Comment:
You need to include the link to your spec change in this PR.  You also need to update generate.cmd to refer to your spec by the latest commit, rather than by the branch name.